### PR TITLE
add boa

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/boa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - conda-mambabuild = boa.cli.mambabuild:main
+    - boa = boa.cli.boa:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pyyaml
+    - colorama
```
Comment (beckermr) [inline recipes/boa/meta.yaml:26]: Are pyyaml and colorama actually needed in host to install the package? I also think you don't need to list setuptools here.

File: recipes/boa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  run:
```
Comment (chrisburr) [inline recipes/boa/meta.yaml:27]:
```suggestion
  run:
    - python >=3.6
```

File: recipes/boa/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  imports:
+    - boa
+    - boa.cli
+  commands:
+    - boa -h
+
+about:
+  home: https://github.com/TheSnakePit/boa
+  license: BSD-3-Clause
+  license_family: BSD
```
Comment (chrisburr) [inline recipes/boa/meta.yaml:45]: It doesn't really matter but this repo data is now automatically generated from SPDX identifiers
```suggestion
```

File: recipes/boa/meta.yaml (hunk 1 above)
Comment (wolfv) [reply recipes/boa/meta.yaml]: let's see!

**Comment by wolfv** [comment]
It's not yet in the release, will be added with the next one.

**Comment by wolfv** [comment]
done! win failure is unrelated (this is a noarch package anyways)

**Pull request merged**